packages/quantic/force-app/main/default/lwc/quanticRecentQueries/quanticRecentQueries.js

```diff
@@ -0,0 +1,83 @@
+import { LightningElement, api, track } from 'lwc';
+import { registerComponentForInit, initializeWithHeadless } from 'c/quanticHeadlessLoader';
+import { getItemFromLocalStorage, setItemInLocalStorage } from 'c/quanticUtils';
+import recentQueriesLabel from '@salesforce/label/c.quantic_RecentQueries';
+import emptyListLabel from '@salesforce/label/c.quantic_EmptyRecentQueriesLabel';
+
+export default class QuanticRecentQueries extends LightningElement {
```

Comment:
Like we did for the RecentResultsList, this should be renamed to `QuanticRecentQueriesList`